apps/rebar/src/rebar_packages.erl

```diff
@@ -246,7 +246,8 @@ update_package(Name, RepoConfig=#{name := Repo}, State) ->
     ?DEBUG("Getting definition for package ~ts from repo ~ts",
            [Name, rebar_hex_repos:format_repo(RepoConfig)]),
     try r3_hex_repo:get_package(get_package_repo_config(RepoConfig), Name) of
-        {ok, {200, _Headers, Releases}} ->
+        {ok, {200, _Headers, Package}} ->
+            #{releases := Releases} = Package,
```

Comment:
Related to https://github.com/erlang/rebar3/pull/2898/files#diff-2ebcc593695219147a2f7c69c5818191a207270f2bd292f3471790a4f5cab411L69-R116

Reply:
Nice catch :)